docs/testrunner-toolkit/configuration/xcuitest.md

```diff
@@ -1,190 +1,468 @@
 
-__Description__: Details specific to the `xcuitest` configuration.
+## `apiVersion`
+<p><small>| REQUIRED | STRING |</small></p>
 
-__Type__: *object*
+Identifies the version of `saucectl` that is compatible with this configuration.
 
-__Example__:
 ```yaml
-xcuitest:
-  app: ./apps/xcuitest/SauceLabs.Mobile.Sample.XCUITest.App.ipa
-  testApp: ./apps/xcuitest/SwagLabsMobileAppUITests-Runner.app
+apiVersion: v1alpha
 ```
+---
 
-### `app`
+## `kind`
+<p><small>| REQUIRED | STRING/ENUM |</small></p>
 
-__Description__: Path to the application. It supports expanded enviornment variable.
+Specifies which framework is associated with the automation tests configured in this specification.
 
-__Type__: *string*
+```yaml
+kind: xcuitest
+```
+---
+
+## `sauce`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The parent property containing all settings related to how tests are run and identified in the Sauce Labs platform.
 
-__Examples__:
 ```yaml
-  app: ./apps/xcuitest/calc.apk
+sauce:
+  region: eu-central-1
+  metadata:
+    name: Testing XCUITest Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $X_COMMIT_SHORT_SHA
+  concurrency: 5
 ```
+---
+
+### `region`
+<p><small>| OPTIONAL | STRING/ENUM |</small></p>
+
+Specifies through which Sauce Labs data center tests will run. Valid values are: `us-west-1` or `eu-central-1`.
 
 ```yaml
-  app: $APP
+  region: eu-central-1
+```
+---
+
+### `metadata`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The set of properties that allows you to provide additional information about your project that helps you distinguish it in the various environments in which it is used and reviewed, and also helps you apply filters to easily isolate tests based on metrics that are meaningful to you, as shown in the following example:
+
+```yaml
+metadata:
+  name: Testing XCUITest Support
+  build: RC 10.4.a
+  tags:
+    - e2e
+    - Android
+    - beta
+    - featurex
 ```
+---
 
-### `testApp`
+### `concurrency`
+<p><small>| OPTIONAL | INTEGER |</small></p>
 
-__Description__: Path to the testing application. It supports expanded enviornment variable.
+Sets the maximum number of suites to execute at the same time. If the test defines more suites than the max, excess suites are queued and run in order as each suite completes.
 
-__Type__: *string*
+:::caution
+Set this value to equal or less than your Sauce concurrency allowance, as setting a higher value may result in jobs dropped by the server.
+:::
 
-__Examples__:
 ```yaml
-  testApp: ./apps/xcuitest/calc-success.apk
+  concurrency: 5
 ```
 
+Alternatively, you can override the file setting at runtime by setting the concurrency flag as an inline parameter of the `saucectl run` command:
+
+```bash
+saucectl run --ccy 5
+```
+---
+
+### `tunnel`
+<p><small>| OPTIONAL | OBJECT | <span class="highlight sauce-cloud">Sauce Cloud only</span> |</small></p>
```

Comment:
Is the `<span class="highlight sauce-cloud">Sauce Cloud only</span>` making sense as Espresso is only supported on Sauce Cloud ?


```suggestion
<p><small>| OPTIONAL | OBJECT |</small></p>
```